manual/new-starters.md

```diff
@@ -15,6 +15,10 @@ Clone the git-secrets [repository](https://github.com/awslabs/git-secrets) to yo
 Run `sudo make install`.
 This should place git-secrets into your PATH, but if not you may need to move it and run `sudo make install` again OR include the folder made by running `sudo make install` into your PATH.
 
+Then run the following commands to add additional rules to check for AWS account numbers:
+* `git-secrets --add --global '([^0-9])*[0-9]{12}([^0-9])*`: this will add a check for any 12 digit number.
```

Comment:
Just to check - have you run this on all the TDR repos to check that false positives aren't a problem in our code?

Reply:
Run against the 31 TDR repos I have locally. Not sure that is all the TDR repos we have now. Didn't get any false positives on these.